nx.json

```diff
@@ -61,12 +116,170 @@
     },
     "@nx/vite:test": {
       "cache": true,
-      "inputs": ["default", "^production"],
+      "inputs": ["test-vitest-inputs"],
       "options": {
         "passWithNoTests": true,
         "watch": false
       }
     },
```

Comment:
I had to keep the executor defaults to keep e2e passing. ATM I have NO (uppercase intentional) clue why exactly, just some next things to verify. I already added this to the followup todos, just to keep a note here..